website/docs/components/link/standalone/partials/content/content.md

```diff
@@ -0,0 +1,21 @@
+## General recommendations
+
+- Use language consistently within each product (e.g., if “Edit” is used on one page, use that same convention throughout, instead of a similar word such as "Change").
```

Comment:
```suggestion
Use language consistently within each product (e.g., if “Edit” is used on one page, use that same convention throughout, instead of a similar word such as "Change").
```